chapter_linear-regression/synthetic-regression-data.md

```diff
@@ -189,12 +189,11 @@ Beyond that, we set `batch_size` in the built-in data loader
 and let it take care of shuffling examples  efficiently.
 
 :begin_tab:`jax`
-JAX is laser-focused on program transformations and
-accelerator-backed NumPy, so it doesn’t include data loading or munging
-in the JAX library. There are already a lot of great data loaders out
-there, and jax suggests using them instead of reinventing anything.
-Here we'll grab PyTorch’s data loader, and modify it slightly to make
-it work with NumPy arrays.
+JAX is all about NumPy like API with device acceleration and the functional
+transformations, so atleast the current version doesn’t include data loading
+methods. With other  libraries we already have great data loaders out there,
+and JAX suggests using them instead. Here we will grab PyTorch’s data loader,
+and modify it slightly to make it work with JAX.
```

Comment:
"Here we will grab PyTorch’s data loader, and modify it slightly to make it work with JAX."

Are we using/modifying PyTorch's data loader here?

Reply:
Yes, we use a custom `collate_fn` and a custom dataset class for JAX which is built on top of pytorch's dataset and dataloader.